File: js/Power_of_Four.js
/**
 * @param {number} num
 * @return {boolean}
 */
var isPowerOfFour = function(num) {
    while (num > 1 && num % 4 === 0) {
        num = num / 4;
    }
    return num === 1;
};

console.log(isPowerOfFour(5))

File: go/Power_of_Four.go
package main

import "fmt"

func isPowerOfFour(num int) bool {
    for num > 1 && num % 4 == 0 {
        num /= 4
    }
    return num == 1
}

func main() {
    fmt.Println(isPowerOfFour(5))
}
